test/node/download-tracker-magnet.js

```diff
@@ -15,7 +16,7 @@ test('Download using HTTP tracker (via magnet uri)', function (t) {
 })
 
 function magnetDownloadTest (t, serverType) {
-  t.plan(10)
+  t.plan(11)
```

Comment:
This test was not passing with an empty store, I assume this is the good number.